### PR TITLE
Add instructions to pull upstream before build

### PR DESCRIPTION
**Pull request by rsthomp**
This fixes the symbolic links in the build-helper folder, which otherwise keep the helpers from building.

### CODE REVIEW

File: docs/build.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ Install packages:
 
 You will need to set up a sbuild chroot. (See https://github.com/sipb/homeworld/issues/80)
 
+# Pull required libraries and images from upstream
+    $ cd //building
+    $./pull-upstream.sh
```
Comment (celskeggs) [inline docs/build.md]: This is missing a space between the `$` and the `.`.

**Comment by celskeggs** [comment]
These need to be signed with your key instead of GitHub's key.

**Comment by celskeggs** [comment]
Looks good, but I need to verify your GPG key in person.

File: building/build-helpers/helper-go/.gitignore [hunk 2]
```diff
@@ -1 +1 @@
-go
+go*
```
Comment (andersk) [inline building/build-helpers/helper-go/.gitignore]: This should be more specific so as not to match `go1.4-bootstrap-20170531.tar.xz` and `go1.8.3.src.tar.xz` which are already in the repository as symlinks:

```
/go*/
```

(Leading slash: only match immediate children of this directory `helper-go`. Trailing slash: only match those that are themselves directories.)

Comment (celskeggs) [reply building/build-helpers/helper-go/.gitignore]: I'm addressing this in a separate commit for the sake of moving things along.

Comment (celskeggs) [reply building/build-helpers/helper-go/.gitignore]: I'm addressing this in a separate commit for the sake of moving things along.

**Pull request merged**